### ISSUE TITLE
Streamline Channel Structure

### ISSUE THREAD

**Issue by jpb5013** [ProgressiveCoders/functions#113]
**Description**: Standardize pinned items in ProgCode community and functional channels

**Problem**: Pinned channel items are currently very inconsistent, making it hard to find critical information

**Function**: Impacts all ProgCode functions and community channels, carried out by operations team. Note that project channels and channels created by community members are not in scope, however we should still inform the community as an optional best practice to adopt

**Benefit**: Standard information and format in channels will make it easier

**Plan**:

1. Create template pinned items in #operations channel first
- Message with short channel description including purpose
- Channel plan document with objectives, goals, strategy
- Link to respective GitHub
- Link to visual asset 
2. Capture channel feedback in #operations to determine if pinned items provide necessary information
3. Update all functional and staff-created community channels with recommended pinned items:
4. Update [Operations Wiki](https://github.com/ProgressiveCoders/functions/wiki/Operations) with standard process following pilot

Create standard

**Comment by dave-mahler** [comment]
@jpb5013 this also syncs nicely with the work @rthbound and I are doing on #86 once we have airtable automatically updating with key info about each channel, it will make the channel purpose and details more relevant as people scan the channel list

**Comment by jpb5013** [comment]
Cool. If you wanna aggregate the ask here I can close out this issue.

**Comment by dave-mahler** [comment]
If we're standardizing, then does it make sense to use pinned items or should we just make the "Channel Details" larger? Benefit of adding to channel details is that airtable's channel list will automatically update soon, and anyone viewing the channel list will be able to view channel details.

**Comment by jpb5013** [comment]
I like the idea, only challenge is it seems channel details are terrible at
opening links from mobile. I always have to description, copy it, then
paste it. Pinned are also my go to place.

Think we could standardize to one link in the pinned section which
aggregates the key things, perhaps pointing to Airtable?

**Comment by rcscastillo** [comment]
Call and structure, conversation with Ellen the other day that's a great next step. What they've used. 

Ellen: I had a quick chat with Aidan who's open to speak with us. Ana – moderator with Ana from Stack Overflow – once I've made the contact, our desire is to have a convo for the date. Conscious to have this core non-partisan. 

**We need to set a date for a next meeting, coordinate with them. Sunday or next monday.**

Ellen to talk with Joe further!

**Comment by dave-mahler** [comment]
@jpb5013 I think the issue with Airtable for something like this is the edit access. So people could add things, but they lose a lot of the ability to edit. 

Actually, given that we have a relationship with Slack, and they seem generally open to feedback, think we could bring up the mobile link issue and see whether they have a fix on the horizon?

**Comment by jpb5013** [comment]
Good point, @dave-Mahler, definitely a challenge. I wonder if a bot or something could enable people to edit through Slack via commands.

And good idea on chatting with Slack. I think if we did it might be good to bundle up the various need (e.g., @channel notifications) into a bigger ask. We've sent some things to their social though I don't think anything's been addressed.
